src/explore-education-statistics-frontend/src/components/DownloadExcelButtonWithAnalytics.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import DownloadExcelButton, {
+  DownloadExcelButtonProps,
+} from '@common/modules/table-tool/components/DownloadExcelButton';
+import { logEvent } from '@frontend/services/googleAnalyticsService';
+
+const DownloadExcelButtonWithAnalytics = (props: DownloadExcelButtonProps) => {
```

Comment:
No need for new '...WithAnalytics' components, as we only need to use them in one or two places. Just add the `onClick` handler to the required pages.